### ISSUE TITLE
missing creaky vowels in clpa

### ISSUE THREAD

**Issue by LinguList** [glottobank/clpa#29]
this thread is used to collect a number of creaky vowels missing in clpa, it unites several one-vowel issues, as  #26, #25, etc.

**Comment by LinguList** [comment]
List of missing vowels:

* ŏ̱ 
* e̱ 
* ĕ [not creaky, rather ultra-short, should be there, so this is a case for aliases, due to different unicode!]
* i̱ 
* o̱ 
* u̱ 
* a̱ 
* ɪ̆  [ultrashort, see above!]

**Comment by nh36** [comment]
Here are some other creaky vowels that come up. 
ø̱  ə̱ ɑ̱̆ i̱ ɔ̱̃ ɛ̱ ɛ̱̃ ɔ̱

**Comment by LinguList** [comment]
We define vowels in clpa by using simple categories as outlined in [this file](https://github.com/glottobank/clpa/blob/master/pyclpa/data/clpa.test.tsv). This means, if a certain variety is "creaky", we would add "creaky" as a feature to the existing quality, thus:

* v027	u	u	vowel	back_rounded	high	plain	plain	868

would become 

* v017 	ṵ 	ṵ 	vowel 	back_rounded 	high 	plain 	creaky

Note that the under**stroke** in the symbols you give is potentially leading to the wrong mapping to clpa, as clpa uses the little tilde beyond the vowel.

**Comment by nh36** [comment]
I am putting a̰  etc. into the Burmish spreadsheet then.

**Comment by LinguList** [comment]
yep! I'll still keep the issue open, as not ALL of the vowels are really
in clpa yet, but this is then up to clpa, not to burmish.
